reg_tests/global_cycle/C192.lndincsoilnoahmp.sh

```diff
@@ -22,13 +22,13 @@ export CDATE=2019073000
 export FHOUR=00
 export DELTSFC=6
 
-export CASE=C768
+export CASE=C192
 
-export COMIN=$HOMEreg/input_data
-export FNTSFA=$COMIN/gdas.t00z.rtgssthr.grb
-export FNSNOA=$COMIN/gdas.t00z.snogrb_t1534.3072.1536
-export FNACNA=$COMIN/gdas.t00z.seaice.5min.blend.grb
-export NST_FILE=$COMIN/gdas.t00z.dtfanl.nc
+export COMIN=$HOMEreg/input_data_noahmp
```

Comment:
You can remove these commented out lines.

Reply:
Done.